index.html

```diff
@@ -3,9 +3,31 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta main="keyword " content="html, Code the Dream">
+    <meta name="author" content="TomN">
     <title>TomN's Portfolio</title>
 </head>
 <body>
-    
+    <h1>Tom Namhay</h1>
+    <section id="About">
+        <h2>About</h2>
+        <!-- <img src="img_chania.jpg" alt="Flowers in Chania"> -->
+    </section>
+    <p>I'm student in CTD</p>
+    <section id="Experience"> 
+        <h2 >Experience</h2>
+    </section>
+    <ul>
+        <li>1</li>        
+        <li>2</li>
+        <li>3</li>
+    </ul>
```

Comment:
They want this list inside your experience section, like this:
```
<section
    <h2
    <ul
        <li
        <li
        <li
    </ul
</section
```

I think they do actually want you to examples of your own experience instead of 1, 2, and 3, because you are actually making a real portfolio to put on your future resume. :)